Makefile

```diff
@@ -3,6 +3,11 @@ IMAGE=miry/online_games_bot
 BUILD_NAME=$(IMAGE):$(VERSION)
 LOG_LEVEL=debug
 SHELL=bash
+NAME=
+CPATH=
+EXTENSION=
```

Comment:
@Ftalysh Can you give me some examples what are possible values for this varibale?